Fix inspect output for falsey values

zokioki opened the pull request:
This change ensures that falsey values are reflected correctly when printing
out the final result - they currently print out as `nil`. For example:

```ruby
class MyStoreModel < StoreModel
  attribute :one, :integer, default: 1
  attribute :two, :boolean, default: false
  attribute :three, :string, default: nil
end
```

Current Behavior: `<MyStoreModel one: 1, two: nil, three: nil>`
Expected Behavior:  `<MyStoreModel one: 1, two: false, three: nil>`

DmitryTsepelev commented:
Hi @zokioki, thanks for the PR, it looks great! Could you please add a test for that (to make sure we won't have a regression in future) and add yourself into the changelog?

zokioki commented:
@DmitryTsepelev Thanks for a useful library! Just updated the specs to reflect nil and boolean attributes, updated the changelog and bumped the version as well. Let me know if that looks good.

DmitryTsepelev commented on CHANGELOG.md:
```diff
@@ -57,5 +61,5 @@
 - Initial version ([@DmitryTsepelev][])
 
-[@DmitryTsepelev]: https://github.com/DmitryTsepelev
+[@dmitrytsepelev]: https://github.com/DmitryTsepelev
```
please switch this line back

zokioki replied:
weird, looks like one of my linters tweaked this - reverted via https://github.com/DmitryTsepelev/store_model/pull/38/commits/a8d9cb3d437b6aedbbd88c6027227d9885c46ad0.

DmitryTsepelev commented on CHANGELOG.md:
```diff
@@ -2,6 +2,10 @@
 
 ## master
 
+## 0.5.4 (2019-09-17)
```
I'll bump the version by myself so let's remove this line

DmitryTsepelev commented on lib/store_model/version.rb:
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module StoreModel # :nodoc:
-  VERSION = "0.5.3"
```
please revert, I'll bump version by myself
